### PR TITLE
LITE-19978: Validate EAAS Runner version

### PR DESCRIPTION
**Pull request by jpdolz**

### CODE REVIEW

File: connect/cli/plugins/project/extension_helpers.py [hunk 1]
```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
...
 
+    _runner_version_validation(project_dir)
+
     click.secho(f'Extension Project {project_dir} has been successfully validated.', fg='green')
 
 
+def _runner_version_validation(project_dir):
+    res = requests.get(PYPI_EXTENSION_RUNNER_URL)
+    if res.status_code == 200:
+        latest_version = res.json()['info']['version']
```
Comment (marcserrat) [inline connect/cli/plugins/project/extension_helpers.py:107]: i will feel more confortable that you ensure that is really a json......i saw it tons of times that systems returns valid code but not what is expected (for example under maintenance situations)

File: connect/cli/plugins/project/extension_helpers.py [hunk 2]
```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
...
+    else:
+        raise ClickException(
+            'We can not retrieve the current connect-extension-runner version from PYPI platform.'
+            '\nPlease check manually if the version on `docker-compose.yml` file is the same than the lastest on PYPI.',
```
Comment (marcserrat) [inline connect/cli/plugins/project/extension_helpers.py:111]: maybe also add where (url) in PyPi such data can be obtained?

File: connect/cli/plugins/project/extension_helpers.py [hunk 3]
```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
...
+        )
+    docker_compose_file = os.path.join(project_dir, 'docker-compose.yml')
+    if not os.path.isfile(docker_compose_file):
+        raise ClickException(
+            f'There is no `docker-compose.yml` file on directory `{project_dir}`, have you deleted it?'
```
Comment (marcserrat) [inline connect/cli/plugins/project/extension_helpers.py:116]: I think here just a message saying that mandatory docker-compose.yml file is needed...rest sounds too colleague style ;)

**Review by vlas — APPROVED** [review]

File: connect/cli/plugins/project/extension_helpers.py [hunk 4]
```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
...
+            '\nWithout a `docker-compose.yml` file the extension development will be harder. '
+            'Please, consider to recover it.',
+        )
+    try:
+        with open(docker_compose_file, 'r') as fp:
+            data = yaml.load(fp, Loader=yaml.FullLoader)
+    except yaml.YAMLError as error:
+        raise ClickException(
+            'The file `docker_compose.yml` is not well configured. Please, consider to fix it.'
```
Comment (marcserrat) [inline connect/cli/plugins/project/extension_helpers.py:125]: `docker_compose.yml`file is not properly formated. Please review it.

**Review by marcserrat — CHANGES_REQUESTED** [review]

File: connect/cli/plugins/project/extension_helpers.py (hunk 1 above)
Comment (jpdolz) [reply connect/cli/plugins/project/extension_helpers.py]: Done!!!
Now the data retrieved is checked...just in case ;-)

**Pull request merged**